Codecademy/gamut
# feat: Add office building double icon

### Overview

Needed for new "For teams" mobile appheader icon, see [figma](https://www.figma.com/file/sDUwae8iT3v72SBkkjxPVq/Career-center-links-on-site?type=design&node-id=78-1393&mode=design&t=1CBClVRNoQ1UQQ1l-0)





### PR Checklist

- [x] Related to designs: https://www.figma.com/file/sDUwae8iT3v72SBkkjxPVq/Career-center-links-on-site?type=design&node-id=78-1393&mode=design&t=1CBClVRNoQ1UQQ1l-0
- [x] Related to JIRA ticket: [REACH-3116](https://codecademy.atlassian.net/browse/REACH-3116)



### Testing instructions

- open storybook preview
- navigate to icons -> regular
- scroll down and ensure OfficeBuildingDoubleIcon appears in the list and looks correct




[REACH-3116]: https://codecademy.atlassian.net/browse/REACH-3116?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
